controllers/subscription/subscription_controller.go

```diff
@@ -241,3 +248,32 @@ func (r *SubscriptionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&operatorsv1alpha1.Subscription{}).
 		Complete(r)
 }
+
+func (r *SubscriptionReconciler) allowDatabaseUpdates(ctx context.Context, installation *integreatlyv1alpha1.RHMI, isServiceAffecting bool) error {
+	if installation.Status.ToVersion != "" && isServiceAffecting {
+		postgresInstances := &crov1alpha1.PostgresList{}
+		if err := r.Client.List(ctx, postgresInstances); err != nil {
+			return fmt.Errorf("failed to list postgres instances: %w", err)
+		}
+		for _, pgInst := range postgresInstances.Items {
+			pgInst.Spec.AllowUpdates = true
```

Comment:
@adam-cattermole  seems the name should be updated to new one (?)

Reply:
Sure, this won't compile or run until 0.41.0 release, at which point it can be rebased to incorporate the new CRO api changes.